docs/reference/inference/put-inference.asciidoc

```diff
@@ -67,10 +67,10 @@ Click the links to review the configuration details of the services:
 * <<infer-service-elasticsearch,Elasticsearch>> (`rerank`, `sparse_embedding`, `text_embedding` - this service is for built-in models and models uploaded through Eland)
```

Comment:
```suggestion
* <<infer-service-elasticsearch,Elasticsearch>> (`chat_completion`, `rerank`, `sparse_embedding`, `text_embedding` - this service is for built-in models and models uploaded through Eland)
```

Reply:
As we chatted about offline, the `elasticsearch` service is different from the `elastic` service. So this code will actually stay the same. We'll need to add a new entry for the `elastic` service.